src/main/docker/docker-compose.yml

```diff
@@ -1,21 +1,11 @@
-version: '2.4'
```

Comment:
You should specify a version here.

Reply:
We aimed for the most recent version which merges v2 and v3 and apparently does not have a version number :D https://docs.docker.com/compose/compose-file/compose-versioning/#versioning